data-raw/roak_ex.R

```diff
@@ -0,0 +1,31 @@
+# Title: `roak` example data sets
+# Description: Data sets provided during the initial development of sdtm.oak
+# by the team at Roche.
```

Comment:
Can we remove {roak} references in {sdtm.oak}. Users may not understand the purpose of this.

Reply:
All source files in `data-raw/` directory are not available to users. So they won't be confused unless they read the source code of the package. Those three lines you mention are only for developers' own reference.